subprojects/packagefiles/iir1/iir/meson.build

```diff
@@ -0,0 +1,41 @@
+sources = files(
+  'Biquad.cpp',
+  'Butterworth.cpp',
+  'Cascade.cpp',
+  'ChebyshevI.cpp',
+  'ChebyshevII.cpp',
+  'Custom.cpp',
+  'PoleFilter.cpp',
+  'RBJ.cpp')
+
+cpp_args = []
+cxx = meson.get_compiler('cpp')
+host_cpu_family = host_machine.cpu_family()
+
+# Maybe disable exceptions
+if not get_option('exceptions')
+  cpp_args += '-DIIR1_NO_EXCEPTIONS'
+endif
+
+# Set commmonly available SIMD flags
+if host_cpu_family.startswith('x86')
+  cpp_args += ['-mfpmath=sse', '-msse2']
+elif host_cpu_family == 'arm'
+  cpp_args += ['-mfpu=neon-vfpv4',
+               '-mfloat-abi=hard',
+               '/arch:VFPv4']
+elif host_cpu_family == 'aarch64'
+  cpp_args += '-mfpu=neon-fp-armv8'
+endif
+# Note: MSVC by-default uses SSE2 instructions (unless overridden)
+# https://docs.microsoft.com/en-us/cpp/build/reference/arch-x86
+
+add_project_arguments(cxx.get_supported_arguments(cpp_args),
+                      language: 'cpp')
+
+lib = library('iir1', 
```

Comment:
The upstream library name is "iir", no 1 at the end.

There's also the cmake equivalent of Meson's `version: meson.project_version()`.

Reply:
Renamed.
Added creation of simple pkgconfig and cmake records.

Reply:
The library still needs to have a `version:` kwarg set.

(I would not bother with the cmake record, since cmake has several different files you need to install, and just the version file won't be enough to get things working. Luckily the pkg-config file is super simple and stands complete on its own!)

Reply:
Added.